int32 sensor_id
geometry_msgs/Point position3d
float32 roll
float32 pitch
float32 yaw